packages/relationships/src/lib/family/index.ts

```diff
@@ -0,0 +1,7 @@
+import { Graph, Edge, RelationshipEvent } from '../../utils/graph.types';
+
+const mapFamily = (graph: Graph, events: Array<RelationshipEvent>): Array<Edge> => {
+  return [];
```

Comment:
I take it actual relationship mapping will come in a later PR?

Reply:
That was the plan